src/pages/[platform]/build-a-backend/server-side-rendering/nuxt/index.mdx

```diff
@@ -273,7 +270,7 @@ export default defineNuxtPlugin({
     );
 
     // Create the libraryOptions object
-    const libraryOptions: LibraryOptions = {
+    const libraryOptions = {
```

Comment:
is this type available from the main package?

Reply:
@josefaidt This type is not available in the main package. In this example it's not necessary, since `tokenProvider` and `credentialsProvider` are typed individually